safe/gui/tools/batch/batch_dialog.py

```diff
@@ -402,7 +402,7 @@ def define_layer(self, layer_path):
             else:
                 LOGGER.critical('Failed to create layer from scenario')
                 return
-        elif extension in ['.shp', '.geojson', '.gpkg']:
+        elif extension in ['.shp', '.json', '.geojson', '.gpkg']:
```

Comment:
Yes +1 ;-)
It would be much better.
https://github.com/inasafe/inasafe/blob/develop/safe/datastore/datastore.py#L153